prac_06/guitars.py

```diff
@@ -0,0 +1,30 @@
+"""
+CP1404/CP5632 Practical
+Guitar Test
+Estimate: 20 minutes
+Actual:   26 minutes
+"""
+from guitar import Guitar
+
+guitars = []
+name = input('Name:')
+while name != '':
+    year = int(input('Year:'))
+    cost = float(input('Cost:'))
```

Comment:
Did you consider what currency the cost should be?
Also maybe consider adding a space after the colon to make it look neater?